src/nrnpython/nrnpy_nrn.cpp

```diff
@@ -1212,7 +1233,13 @@ static PyObject* pysec_wholetree_safe(NPySecObj* const self) {
 static PyObject* pysec2cell(NPySecObj* self) {
     nb::object result;
     if (self->cell_weakref_) {
+#if PY_VERSION_HEX >= 0x030D0000
+        PyObject* cell = nullptr;
+        PyWeakref_GetRef(self->cell_weakref_, &cell);
+        result = nb::steal(cell);
```

Comment:
Just to clear it up for myself: since `PyWeakref_GetRef` gets a "strong reference", that should increase the refcount of `cell`, so this is why we `steal` it instead of `borrow`, right?

Reply:
right